src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/CngAlgorithm.cs

```diff
@@ -219,6 +219,17 @@ public static CngAlgorithm Sha512
         [Experimental(Experimentals.PostQuantumCryptographyDiagId, UrlFormat = Experimentals.SharedUrlFormat)]
         public static CngAlgorithm MLKem => field ??= new CngAlgorithm("ML-KEM"); // BCRYPT_MLKEM_ALGORITHM
 
+        /// <summary>
+        ///   Gets a new <see cref="CngAlgorithm"/> object that specifies the Stateless Hash-based Digital Signature
```

Comment:
```suggestion
        ///   Gets a new <see cref="CngAlgorithm"/> object that specifies the Stateless Hash-Based Digital Signature
```
if we want to follow ML-DSA